mongodb/intro/date-aggregation/aggregation-with-date-expression-operators-ii.md

```diff
@@ -0,0 +1,162 @@
+---
+author: Stefan-Stojanovic
+
+aspects:
+  - introduction
+  - new
+
+type: normal
+
+category: how to
+
+---
+
+# Aggregation With `$year`, `$month` and `$dayOfMonth`
+
+---
+## Content
+
+Now that we know how to extract the date into an ISODate format, we can use some aggregation operators to find the exact time that the document was made as precise as in milliseconds.
+
+**Note:** For easier readability and comprehension of date operators, we will be using the date from the `name:"Bulbasaur"` document shown in the previous insight. **(The document will also be in the footnotes of every insight of this workout under Previous Document[1])**
+
+**Note:** All 10 date operators have the same syntax:
+
+```js
+{
+  $DATE_OPERATOR: <expression>
+}
+```
+
+### $year
+
+The `$year` operator is used to find the year the document was created.
+
+Since we are only looking for the date of a specific document, we will use the `$match` stage to only match the document we want. Also, we will use the `$project` stage to only aggregate the parts of the date we want.
```

Comment:
I'm not sure "aggregate" is the right word here. how does this sound?
```suggestion
Since we are only looking for the date of a specific document, we will use the `$match` stage to only match the document we want. We'll use the `$project` stage to display only the parts of the date we want.
```